docs/Validation.md

```diff
@@ -99,7 +99,7 @@ These tests are performed when a forecast is created or updated.
 
 ## Tests for Prediction Elements by Target Type
 
-These tests are performed when a forecast is created or updated. For all target types, only [valid Prediction Types](Targets.md#valid-prediction-types-by-target) are accepted.
+These tests are performed when a forecast is created or updated. For all target types, only [valid Prediction Types](Targets.md#valid-prediction-types-by-target) are accepted. Note: `point` also applies to `mean`, `median`, and `mode`.
```

Comment:
I suggest making these changes explicitly in the text below, not just once at the top. Will save us work later and be cleaner now.